internal-docs/engineering/environment-variables.md

```diff
@@ -26,4 +26,6 @@ Full list of Environment variables and where they are needed.
 | CYPRESS_RECORD_KEY                 | Cypress record key so we can record our cypress runs                  | ✅    | ✅         |               |               |
 | PERCY_TOKEN                        | Percy.io token                                                        | ✅    | ✅         |               |               |
 | USE_CYPRESS_VRT                    | Key to enable the cypress vrt integration                             | ✅    |            |               |               |
-| NX_CLOUD_ACCESS_TOKEN             | Token to enable nx cloud cache for builds                             | ✅    | ✅         | ✅            | ✅            |
+| NX_CLOUD_ACCESS_TOKEN              | Token to enable nx cloud cache for builds                             | ✅    | ✅         | ✅            | ✅            |
+| OPENAI_API_KET                     | ChatGPT4 API Key                                                      | ✅    |            | ✅            |               |
```

Comment:
I added these to netlify so they should be installed locally upon following the instructions in our docs for developing locally